SwanFileBrowser/src/index.ts

```diff
@@ -0,0 +1,1181 @@
+// Copyright (c) SWAN Development Team.
+// Modified from the original JupyterLab extension.
+// Author: Omar.Zapata@cern.ch 2021
+
+/**
+ * SwanFileBrowser extension is based on the upstream filebrowser extension 3.0.x.
+ *
+ * This file is the entry point for the extension where is defined the commands such as
+ * 'filebrowser:copy', 'filebrowser:cut', 'filebrowser:open' etc..
+ * Those commands are reimplemented using our class SwanFileBrowser instead the default Jupyter FileBrowser.
+ *
+ * The classes SwanFileBrowser and SwanFileBrowserModel are defined in the file swanfilebrowser.ts
+ * and the class SwanDirListing in the file listing.ts. Those have the code required to manipulate
+ * projects and the kernel manager.
+ *
+ * Alsmot all the documentation for this file is the default one provided by the Jupyter extension.
```

Comment:
I think it's worth giving a bit of detail of what you modified in this file. Perhaps you can explain that a hard copy of the original index.ts of the JupyterLab file browser was necessary because it was not possible to just extend it. After that, you can mention what exactly you modified in this long file - this will make it easier for future developers of this code to update it to future versions of the JLab file browser.

Reply:
let me know if this update is good enough for you.